src/game/Tactical/Points.h

```diff
@@ -41,6 +41,10 @@
 #define AP_MOVEMENT_LAKE        9       // deep wade -> slowest
 #define AP_MOVEMENT_OCEAN       8       // swimming is faster than deep wade
 
+#define AP_MOVE_STANDING        2
+#define AP_MOVE_CROUCHING       3
+#define AP_MOVE_PRONING         4
```

Comment:
Wasn't it 6 for prone movement? Also, if you run, standing movement cost is 1.

Reply:
In turn-based modus it's as defined. The ap costs are calculated per tile.

Reply:
You're right about prone, but please double check running — the first tile costs the same (2), the next ones only 1.

Reply:
In turn based there is a fixed ap price for walking over tiles depending on stance. But I don't need these defines anymore so I will stripe them out.

Reply:
Cool, but what I wrote is also true — tested in the original.